### PR TITLE
fixed some notifications / order event stuff

### PR DESCRIPTION
**Pull request by joeykrug**

### CODE REVIEW

**Review by andrewdamelio — APPROVED** [review]
nice!

File: packages/augur-sdk/src/api/ZeroX.ts [hunk 1]
```diff
@@ -329,10 +329,15 @@ export class ZeroX {
...
-      return this.meshClient.addOrdersAsync([order]);
+    try {
+      if (this.browserMesh) {
+        return this.browserMesh.addOrdersAsync([order]);
+      } else {
+        return this.meshClient.addOrdersAsync([order]);
+      }
+    }
+    catch(error) {
+      return setTimeout(this.addOrder(order), 5000);
```
Comment (andrewdamelio) [inline packages/augur-sdk/src/api/ZeroX.ts:340]: 👍

**Review by andrewdamelio — APPROVED** [review]
lgtm

File: packages/augur-sdk/src/api/ZeroX.ts (hunk 1 above)
Comment (pgebheim) [reply packages/augur-sdk/src/api/ZeroX.ts:340]: This needs to actually detect the error and probably do some sort of back-off with a limit or we can get into a terrible busy loop here.

**Pull request merged**